src/peergos/shared/user/fs/FileWrapper.java

```diff
@@ -1111,42 +1145,44 @@ public SigningPrivateKeyAndPublicHash signingPair() {
     }
 
     @JsMethod
-    public CompletableFuture<FileWrapper> copyTo(FileWrapper target, UserContext context) {
+    public CompletableFuture<Snapshot> copyTo(FileWrapper target, UserContext context) {
```

Comment:
this may impact web-ui code. Do we want to expose Snapshot in API?

Reply:
Good call. I've changed it to a boolean